src/backend/parser/parse_partition_gp.c

```diff
@@ -1120,6 +1127,61 @@ merge_partition_encoding(ParseState *pstate, List *elem_colencs, List *penc)
 	return elem_colencs;
 }
 
+static List *
+mergeInheritedAndDefaultEncodings(GpPartDefElem *elem,
```

Comment:
In the file there is a mix of snake and camel cases. But the functions right above and below the new one are in snake case. So I suggest to use it here as well:
```suggestion
merge_inherited_and_default_encodings(GpPartDefElem *elem,
```

Reply:
Updated.